src/main/java/org/jboss/modules/PathUtils.java

```diff
@@ -174,8 +207,9 @@ public static String canonicalize(String path) {
             outer: switch (c) {
                 case '/': {
                     inner: switch (state) {
-                        case 0: state = 3; e = i; break outer;
-                        case 1: state = 3; e = i; break outer;
+                        case 0:
+                        case 1:
+                            state = 3; e = i; break outer;
```

Comment:
I'd prefer to keep this as it was for readability, or at least keep the `case 1` line as it is and change `case 0` to be `case 0: // fall through`. I'd like a similar change to the other two sites.

Reply:
I'll revert this. It's an artifact of me just clicking around and listening to the IDE.